src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -546,6 +546,7 @@ LottieRect* LottieParser::parseRect()
 LottieEllipse* LottieParser::parseEllipse()
 {
     auto ellipse = new LottieEllipse;
+    if (!ellipse) return nullptr;
```

Comment:
@mgrudzinska just for your information, I intentionally removed the null check from the memory alloc fails.

Reply:
@hemet it makes perfect sense, since new will throw ex, not return null

Reply:
I added this, sorry, rebasing didn;t go well